provisioning/requirements.yml

```diff
@@ -1,6 +1,7 @@
 - src: https://github.com/idi-ops/ansible-facts
   name: facts
 
-- src: https://github.com/gtirloni/ansible-role-nodejs
+- src: git+https://github.com/gtirloni/ansible-role-nodejs.git
```

Comment:
Could we have some commenting here? What has changed here, and why? It would help if we could cite the npm thread in which this problem has appeared, and explain how/if our approach is derived from one of the suggestions there